src/api/docgen/transpile/python.ts

```diff
@@ -0,0 +1,323 @@
+import { toSnakeCase } from "codemaker";
+import * as reflect from "jsii-reflect";
+import { Markdown } from "../render/markdown";
+import * as transpile from "./transpile";
+
+/**
+ * A python transpiler.
+ */
+export class PythonTranspile extends transpile.AbstractTranspile {
```

Comment:
We usually use the Base postfix to indicate an abstract base class:

```suggestion
export class PythonTranspile extends transpile.TranspileBase {
```

Reply:
👍